mitxpro/settings.py

```diff
@@ -32,6 +32,11 @@
 
 SECURE_SSL_REDIRECT = get_bool("MITXPRO_SECURE_SSL_REDIRECT", True)
 
+HUBSPOT_CONFIG = {
+    "FORM_GUID": get_string("FORM_GUID", None),
```

Comment:
There will be another hubspot form (for the form in the footer), so we'll need a more specific name for this setting.

Reply:
any suggested name for that form ?

Reply:
For this one, how about `NEW_COURSES_FORM_GUID` and for the other `FOOTER_FORM_GUID` ?